Gemfile

```diff
@@ -6,7 +6,7 @@ gem 'rails_drivers'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', group: :doc
 
-gem 'nokogiri', '>= 1.12.5' # >= 1.11.0.rc4 due to CVE-2020-26247
+gem 'nokogiri', '>= 1.13.4' # >= 1.11.0.rc4 due to CVE-2020-26247
```

Comment:
Should we move to 1.13.6 for [CVE-2022-29181](https://nvd.nist.gov/vuln/detail/CVE-2022-29181) as long as we are reving it? Hopefully, we can remove it shortly.